docs/source/install.rst

```diff
@@ -6,11 +6,11 @@ You can install dask with ``conda``, with ``pip``, or by installing from source.
 Conda
 -----
 
-To install the latest version of Dask from the
-`conda-forge <https://conda-forge.github.io/>`_ repository using
+Binary Dask packages are maintained on both the ``defaults`` and `conda-forge
+<https://conda-forge.github.io/>`_ channels.  You can install them using
 `conda <https://www.continuum.io/downloads>`_::
```

Comment:
OK, I've mentioned Anaconda at the start.  I then immediately say that you can conda install dask.  Afterwards I mention that Dask packages are maintained on conda-forge.  I think that the way this is phrased it will not register much on people who don't understand channels, which is probably good because they probably don't want to use them.